GNUmakefile

```diff
@@ -110,11 +110,14 @@ go-solidity-wrappers-functions: pnpmdep abigen ## Recompiles solidity contracts
 generate: abigen codecgen mockery ## Execute all go:generate commands.
 	go generate -x ./...
 
+
+DIR_ARGS ?= -recurse=true
```

Comment:
Does this need to be global?
I'm actually not sure I understand it. Could you add some comments here with examples?

Reply:
It does if you want default args unfortunately...

I was trying to achieve the following with this:
- make it possible to call the command "naked" and have it run all the txtars
- make it possible to just pass in a directory and have it just run that directory (i.e. not recursively)

There was no way to do that without adding a Make default somewhere.

In any case, the resulting syntax is still very awkward but I wasn't sure when to call quits on improving this further.